model/histogram/float_histogram.go

```diff
@@ -1118,3 +1045,10 @@ func floatBucketsMatch(b1, b2 []float64) bool {
 	}
 	return true
 }
+
+func (h *FloatHistogram) ReduceResolution(targetSchema int32) *FloatHistogram {
```

Comment:
This needs a doc comment (including that the targetSchema must be smaller than the current schema).